data-reconciliation-app/demo-app/demo-guidance.md

```diff
@@ -15,12 +15,12 @@ We have sample sets [here](./data/) for the FSI Demo. Brent will provide real da
 1. Deploy data reconciliation
```

Comment:
Might play havoc with your links - but do you think that this document would be better off in the data-reconciliation-app/docs folder. Then we also slowly delete this demo folder?

Reply:
Agree, but we can do it as seperate PR